### ISSUE TITLE
Build failing when including WPF items

### ISSUE THREAD

**Issue by GeertvanHorrik** [onovotny/MSBuildSdkExtras#66]
Today I had a really weird issue when converting a test project (of Catel) to the new project structure. It was building fine, but once I did a few (seemed simple) changes, the build suddenly broke:

```
PublicApiFacts.cs(11,11): error CS0246: The type or namespace name 'ApiApprover' could not be found (are you missing a using directive or an assembly reference?) [C:\CI_WS\Ws\110208\Source\Catel\src\Catel.Tests\Catel.Tests_fh3rqzzc_wpftmp.csproj]
```

Visual Studio also started complaining that it was unable to see the item, but it was there! Then I decided to remove a test user control I added (to check for xaml resource existence). And suddenly the error disappeared.

Not sure what's happening under the hood, but I think it's weird that adding a WPF user control breaks the build on a totally different item (e.g. a NuGet package with file references that are automatically linked in):

![image](https://user-images.githubusercontent.com/1246444/39587958-0b886a28-4efb-11e8-9b1d-52540dd2a615.png)

**Comment by GeertvanHorrik** [comment]
Just verified, after a working build, just adding 2 files (.xaml and .xaml.cs) breaks the build.

**Comment by GeertvanHorrik** [comment]
It gets even worse, this is the line causing the build to be broken:

![image](https://user-images.githubusercontent.com/1246444/39588305-d58d98e8-4efb-11e8-8906-505b2698d1c7.png)

Not sure whether this is related to MSBuildSdkExtras.

**Comment by onovotny** [comment]
Do you have .NET 4.7.2 installed on your machine? What OS are you using (it's included with Windows 10 1803 or an update for the others).

**Comment by GeertvanHorrik** [comment]
Probably yes, have been running the latest windows insider for a while now.

**Comment by Nirmal4G** [comment]
@onovotny @GeertvanHorrik Use this workaround

https://github.com/nirin/msbuild-sdks/blob/develop/Source/MSBuild.NET.Extras.Sdk/Build/MSBuild.NET.Workarounds.props

https://github.com/nirin/msbuild-sdks/blob/a586fc314765aa8b512ba0b79f9f52ba9b92b43c/Source/MSBuild.NET.Extras.Sdk/Build/MSBuild.NET.Workarounds.targets#L7-L27

**Comment by GeertvanHorrik** [comment]
Thanks for the research. So I think @onovotny should look into your workaround to see if it can be added to this repository, is that correct?

**Comment by Nirmal4G** [comment]
Yes, but I'm already adding along with a bunch of fixes and features as part of #56!

**Comment by GeertvanHorrik** [comment]
Thanks, then I will close this ticket. Thanks for the help.
